runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItem.java

```diff
@@ -38,12 +39,18 @@
 	private String name;
 	private int quantity;
 	private WorldPoint location;
+	private LocalPoint localLoc;
```

Comment:
Ground items are cleared by the plugin whenever a new scene loads, so there is never a reason to store both the world and local location.